assets/templates/detail-fullscreen.html

```diff
@@ -0,0 +1,9 @@
+{{> navbar }}
+
+{{> panel-left }}
+{{> panel-right }}
+{{> overlay fullscreen="true" }}
```

Comment:
For boolean option, you can pass a boolean parameter :wink:  -> `{{> overlay fullscreen=true }}`